CMakeLists.txt

```diff
@@ -1,7 +1,7 @@
 project(MaterialX)
 set(MATERIALX_MAJOR_VERSION 1)
-set(MATERIALX_MINOR_VERSION 36)
```

Comment:
Updated the MaterialX version, was needed for the upgrade path to work correctly.